tests/cypress/e2e/hedy_page/editor_box.cy.js

```diff
@@ -90,4 +94,14 @@ describe('Test editor box functionality', () => {
     cy.get('[data-cy="error_details"] span').should('have.class', 'command-highlighted');
 
   });
-});*/
\ No newline at end of file
+});
+
+/**
+ * Wait for the Ace editor to finish initializing all the HTML elements and attach the listeners
+ * 
+ * If we start typing in the editor immediately, it may happen before the editor has
+ * loaded. Waiting for HTML elements that are created as the editor loads will fix that.
+ */
+function waitForAceToLoad() {
+  cy.get('#editor > .ace_scroller > .ace_content').should('be.visible');
```

Comment:
Yes, I tried this and it didn't work. You can try locally by running the spec in the command line! It works in the UI but not there! Can you try with `--browser chrome` it worked for me locally too.

Reply:
I've been able to repro the failure once on my machine, and then it never failed again 😢

Reply:
It's interesting though that the `Language XX should run` tests never seem to fail, though they do something quite similar.

Reply:
> It's interesting though that the `Language XX should run` tests never seem to fail, though they do something quite similar.

I know right? That's why I was baffled that these tests where failing so much.